code/_onclick/hud/ability_screen_objects.dm

```diff
@@ -60,9 +61,15 @@
 		overlays.Add(open_state)
 	update_icon()
 
-/obj/screen/movable/ability_master/proc/open_ability_master()
+/obj/screen/ability_master/proc/open_ability_master()
+	for(var/i = 1 to length(ability_objects))
+		var/obj/screen/ability/A = ability_objects[i]
+		var/row = round(i/abilities_per_row)
+		A.screen_loc = "RIGHT-[(i-(row*abilities_per_row))+2]:16,TOP-[row+1]:16"
```

Comment:
Would be nice to move this to vis_contents in the future but I'm leaving it as screen_loc for now.